docs/validate-stable-rc-builds.md

```diff
@@ -0,0 +1,237 @@
+# Stable rc review validation process
+Following steps to be performed when a stable rc review pushed on stable mailing
+list
+
+## Wait for stable rc review email
+Greg KH pushes on to stable rc trees and creates a tag
```

Comment:
Actually, no tag is created/published. (That would make things easier for lots of people.)